api/views.py

```diff
@@ -248,6 +249,38 @@ def get_queryset(self):
         return Comment.objects.get(pk=self.kwargs['comment_pk']).commentlike_set.all().order_by('author_id')
 
 
+def handle_inbox_follow(request: Request, body: dict[str, Any]) -> Response:
```

Comment:
Do we need to check for the request method here (post, etc.)?

Reply:
It's always POST according to line 47